src/pages/AdminJobPostings.jsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect } from "react";
+import { useState } from "react";
+
+import database from "@/clients/firebase";
+import { JobView } from "@/components/JobView";
+import { fetchJobs } from "@/functions/fetchJobs";
+import { useAuth } from "@/models";
+
+export default function AdminJobPostings() {
+  const [jobs, setJobs] = useState([]);
+  const [selectedJob, setSelectedJob] = useState(null);
+  const auth = useAuth();
+
+  useEffect(() => {
+    if (auth) {
+      getJobs();
+    }
+  }, []);
+
+  const getJobs = async () => {
+    const jobList = await fetchJobs(database, "admin", null);
+    const filteredJobs = jobList.filter((job) => job.status === "pending");
+    setJobs(filteredJobs);
+    setSelectedJob(filteredJobs.length > 0 ? jobList[0] : null);
+  };
+
+  return (
+    <div className="flex flex-row w-screen h-screen justify-evenly bg-base-300">
+      <div className="flex flex-col w-half items-center overflow-scroll">
+        <h2>
+          <strong>Review Requests ({jobs.length})</strong>
```

Comment:
first, it's good for now. 

for style consistence, it could be better to use `<p className="font-bold"></p>`. the point is to apply  [font weight](https://tailwindcss.com/docs/font-weight) or [font-size](https://tailwindcss.com/docs/font-size) and so like. 

Same for  `<h2>` things

Reply:
Ah I see, yep will do